code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -350,12 +350,12 @@
 	item_state = "bullpup"
 	w_class = ITEMSIZE_LARGE
 	force = 10
-	caliber = "a556"
+	caliber = "a762"

```

Comment:
According to the description, this still uses 5.56 ammo. This is not correct.

Or, personally, I think it'd be better if we switched the Space AK for this thing. Don't change the STS ammo or this Assault Carbine's ammo. Just switch which ones are available to players. This gun being a 7.62 makes no sense anyway, and it doesn't match the sprites provided.